easybuild/easyblocks/generic/intelbase.py

```diff
@@ -325,11 +324,11 @@ def install_step(self, silent_cfg_names_map=None, silent_cfg_extras=None):
 
         if self.install_components:
             if len(self.install_components) == 1:
-                # if ALL or DEFAULTS, no quotes should be used
+                # if only a single components if listed (e.g. ALL, DEFAULTS), no quotes should be used
```

Comment:
may be bizar, but we better take it into account

so, in that case, the check here is broken, and should be

```python
if self.install_components) == ['ALL'] or self.install_components) == ['DEFAULTS']:
```

agreed?